# Defines a handover target
string object_id
geometry_msgs/PoseStamped pose